svo (reboundingsileristracker).xml

```diff
@@ -236,7 +236,7 @@ svo.rt_razed = nil</script>
                 <colorTriggerFgColor>#000000</colorTriggerFgColor>
                 <colorTriggerBgColor>#000000</colorTriggerBgColor>
                 <regexCodeList>
-                    <string>^\w+ takes a long drag off (?:his|her) ?.*pipe\.$</string>
+                    <string>^\w+ takes a long drag off (?:his|her)? ?.*pipe\.$|^A great gurgling sound is heard as \w+ takes a long drag from an enamel bubble pipe\. A series of shining bubbles erupt from the bowl of (?:her|his) pipe\.$</string>
```

Comment:
What do you mean? It is in one trigger. You mean like:

` ^(?:\w+ takes a long drag off|A great gurgling sound is heard as \w+ takes a long drag from an enamel bubble pipe\. A series of shining bubbles erupt from the bowl of) (?:her|his) .*pipe\.$ `

that?

Or you mean disaable multiline trigger and set it in a second box? If that's what you're asking, it already is a multiline trigger and the second line's a relevant lua function.